### PR TITLE
Added a general `startAllStones` and `stopAllStones`

### PR DESCRIPTION
**Pull request by marianopeck**
Added 2 new scripts  and  which allow performing such operations even with a specified OS user.

Dale, I am not sure if these are useful enough for the general purpose users of GsDevKit_home. I just opened this PR because I do need this scripts for myself and maybe they are of help to someone else. I don't mind if they are not integrated. Review them carefully please.

### CODE REVIEW

File: bin/startAllStones [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  if [ -d "$stonePath" ] ; then
+    if [ -e "$stonePath/stone.env" ] ; then
+      # set up stone environment
+      pushd $stonePath >& /dev/null
+        source $stonePath/stone.env
+      popd >& /dev/null
+
+
+
+# start the stone
```
Comment (dalehenrich) [inline bin/startAllStones]: Starting here it looks like the indentation is off ... no need to change this unless you are also planning on conditionally using `-b` with `startStone`

**Comment by marianopeck** [comment]
OK, I pushed another commit. This should address everything. What do you think?

**Comment by marianopeck** [comment]
@dalehenrich can you help me understand what is going on with travis? I would like to have these changes integrated back into `gs_port`. Thanks!

**Comment by dalehenrich** [comment]
@marianopeck when the current tests finish I'll take a look and see what I can figure out

**Comment by dalehenrich** [comment]
It looks like you've got the problems fixed, the one failure is due to an mcz download glitch ... I'll go ahead and merge this shortly ...

**Comment by dalehenrich** [comment]
The run times for the builds seems to have significantly increased some have doubled..... of course I can't tell if the runtime are due to overloaded hosts for travis or not --- 

Comparing the two runs ([pull request](https://travis-ci.org/GsDevKit/GsDevKit_home/builds/195919305) and [gs_portMariano]https://travis-ci.org/GsDevKit/GsDevKit_home/builds/195919260()) it looks like gs_portMariano is overall much slower, than the pull request run, so I'll guess it's due to host overloading on travis ...

**Comment by dalehenrich** [comment]
actually the pull request was slower, but it's the same difference ... host overloading

**Comment by marianopeck** [comment]
Hi @dalehenrich 
Ok... so it indeed looks like the tests did pass now right?  Cool.

**Pull request merged**